CodeAssignment_MVVM_Combine/Sources/Views/Cells/HomeViewController/RepositoryCollectionViewCell.swift

```diff
@@ -0,0 +1,278 @@
+//        )
+//        let startImage = UIImage(
+//            systemName: "star",
+//            withConfiguration: config)?
+//            .withTintColor(.systemGray5, renderingMode: .alwaysOriginal)
+        let starImage = UIImage(systemName: "star")?.withTintColor(.systemGray3, renderingMode: .alwaysOriginal)
+        button.setImage(starImage, for: .normal)
+        return button
+    }()
+    
+    /// starの数を表示するLabel
+    private lazy var starCountsLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 16, weight: .regular)
+        label.textColor = .systemGray
+        return label
+    }()
+    
+    /// 言語ごとに色をつけて表示させるためのView
+    private lazy var languageColorView: UIView = {
+        let view = UIView()
+        view.clipsToBounds = true
+        view.backgroundColor = .systemPink
+        return view
+    }()
+    
+    private lazy var languageNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 16, weight: .regular)
+        label.textColor = .systemGray
+        return label
+    }()
+    
+    // コードベースなので、ここで必須のfuncを実装する
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupUI()
+    }
+    
+    // nibファイルを用いて実装していないことをfatalErrorを用いて明確にする
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    /** Cellレイアウトが確定されたタイミングで呼び出されるので、CellのSubViewのCornerRadiusのような設定が可能
+     - lazy varのクロージャ内ではビューのフレームがまだ決定されていないため、frame.heightを使用しても正しい値を取得できない
+     - そのため、layoutSubViewsをオーバーライドして設定する
+    */
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        userAccessoryView.layer.cornerRadius = userAccessoryView.frame.height / 2
+        userImageView.layer.cornerRadius = userImageView.frame.height / 2
+        languageColorView.layer.cornerRadius = languageColorView.frame.height / 2
+    }
+}
+
+// MARK: - Function and Logics
+extension RepositoryCollectionViewCell {
+    /** Cellのデータの設定を行う
+     - 今後、RepositoriesForViewのModelのデータに基づいて、Cellのデータを確立させる
+    */
+    func configure(with model: Repositories.Repository) {
+        let defaultImage = UIImage(systemName: "person.circle")?.withTintColor(.systemMint, renderingMode: .alwaysOriginal)
+        
+        nameLabel.text = model.name
+        descriptionLabel.text = model.description
+        userNameLabel.text = model.owner.userName
+        starCountsLabel.text = formatNumberToString(model.stargazersCount ?? 0)
+        languageNameLabel.text = model.language
+        
+        if let urlString = model.owner.profileImageString,
+           let url = URL(string: urlString) {
+            // completedに入れないと、常にplaceHolderImageしか表示されない仕組み
+            // completion handlerを用いて、画像のロード中にエラーが発生した場合や、URLが無効な場合にデフォルトの画像を表示する仕組みである
+            userImageView.sd_setImage(with: url, placeholderImage: defaultImage) { [weak self] (image, error, cacheType, url) in
```

Comment:
cacheTypeのようにクロージャー内で使用しない値は`_`で書くことができるよ！
```suggestion
            userImageView.sd_setImage(with: url, placeholderImage: defaultImage) { [weak self] image, error, _, _ in
```

Reply:
@tosh7 
[cae3d63]にて、コード修正しました！ご確認、お願いします！